bom/application/pom.xml

```diff
@@ -4559,78 +4541,12 @@
                     </exclusion>
                 </exclusions>
             </dependency>
-            <dependency>
-                <groupId>io.smallrye.reactive</groupId>
-                <artifactId>smallrye-mutiny-vertx-web-client</artifactId>
-                <version>${smallrye-reactive-utils.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.smallrye.reactive</groupId>
-                <artifactId>smallrye-mutiny-vertx-sql-client</artifactId>
-                <version>${smallrye-reactive-utils.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.smallrye.reactive</groupId>
-                <artifactId>smallrye-mutiny-vertx-db2-client</artifactId>
-                <version>${smallrye-reactive-utils.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.smallrye.reactive</groupId>
-                <artifactId>smallrye-mutiny-vertx-mysql-client</artifactId>
-                <version>${smallrye-reactive-utils.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.smallrye.reactive</groupId>
-                <artifactId>smallrye-mutiny-vertx-pg-client</artifactId>
-                <version>${smallrye-reactive-utils.version}</version>
-            </dependency>
             <dependency>
                 <!-- Used by the Reactive PG Client -->
                 <groupId>com.ongres.scram</groupId>
                 <artifactId>client</artifactId>
                 <version>${scram-client.version}</version>
             </dependency>
-           <dependency>
-                <groupId>io.smallrye.reactive</groupId>
-                <artifactId>smallrye-mutiny-vertx-mail-client</artifactId>
-               <version>${smallrye-reactive-utils.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.smallrye.reactive</groupId>
-                <artifactId>smallrye-mutiny-vertx-redis-client</artifactId>
-                <version>${smallrye-reactive-utils.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.vertx</groupId>
-                <artifactId>vertx-kafka-client</artifactId>
-                <version>${vertx.version}</version>
```

Comment:
What about this one? It was unnecessary?

Reply:
Yes, we are not using it anymore.

Reply:
Actually, we still have a dependency - so better keep it. 
We are going to get rid of it, but one step at a time.